### ISSUE TITLE
CI Should Skip PRs for .md files

### ISSUE THREAD

**Issue by sedouard** [Azure/azure-quickstart-templates#957]

**Comment by singhkay** [comment]
This can be achieved by putting a .ci_skip file in the folder

**Comment by sedouard** [comment]
@singhkay ... putting `.ci_skip` in the folder will permenantly disable ci for the template, no matter what file changes, not just the `.md` file

**Comment by singhkay** [comment]
@sedouard you're right. Is the intent to have code that checks if the PR only contains a .md file, then skip the CI validation?

**Comment by sedouard** [comment]
yup. its on the list of stuff I want to do but I haven't had the bandwidth to knock it out just yet
